build/main-branch-sync/sync-utils/go-upgrade.sh

```
@@ -22,17 +30,27 @@ if [[ -f ${REPO_PATH}/go.mod ]]; then
+    ${GO} mod vendor
+  fi
 fi
 
 if [[ -f ${REPO_PATH}/.ci-operator.yaml ]]; then
   yq '.build_root_image.tag = "go'"${go_version}"'-linux"' -i "${REPO_PATH}/.ci-operator.yaml"
 fi
 
-for file in "${REPO_PATH}"/build/Dockerfile* "${REPO_PATH}"/Dockerfile*; do
+konflux_build_files=""
+if [[ -d ${REPO_PATH}/.tekton ]]; then
+  konflux_build_files=$(yq '. as $pipeline ireduce(""; . + ($pipeline.spec.params[] | select(.name == "dockerfile") | .value))' "${REPO_PATH}/.tekton"/*.yaml | sort -u)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)